parser/declaration_parser_test.go

```diff
@@ -382,3 +382,47 @@ director example client {
 	}
 	assert(t, vcl, expect)
 }
+
+func TestParsePenaltybox(t *testing.T) {
+	input := `// Penaltybox definition
+	penaltybox ip_pbox {
+} // Trailing comment`
+	expect := &ast.VCL{
+		Statements: []ast.Statement{
+			&ast.PenaltyboxDeclaration{
+				Meta: ast.New(T, 0, comments("// Penaltybox definition")),
+				Name: &ast.Ident{
+					Meta:  ast.New(T, 0),
+					Value: "ip_pbox",
+				},
+			},
+		},
+	}
+	vcl, err := New(lexer.NewFromString(input)).ParseVCL()
+	if err != nil {
+		t.Errorf("%+v\n", err)
+	}
+	assert(t, vcl, expect)
+}
```

Comment:
Need test for error when some statements exist in penaltybox declaration.

Reply:
hmm not sure about this case to be honest
I ended up allowing parsing statements inside these blocks but linting them as errors in case they exist. 
what do you think?

Reply:
Ah, it's OK, I agree with you.